data/www/app/src/Beaver/Container.php

```diff
@@ -72,7 +72,7 @@ private function createInstance(string $serviceName)
         }
 
         $constructor = $reflector->getConstructor();
-        if (null === $constructor) {
+        if (null === $constructor || empty($constructor->getParameters())) {
```

Comment:
What is it for ?

Reply:
Avant cet ajout les services avec un constructeur sans parametres pétaient car ils n'avaient pas la clé 'parameters'

$serviceConfig['parameters'] pétait